form.php

```diff
@@ -65,7 +66,6 @@
     $shoe['cost'] = $cost;
     $shoe['image'] = $image;
```

Comment:
Are you actually using this `$shoe` variable anywhere? I think these lines can be removed.